Nodejs/Product/Nodejs/Options/NodejsIntellisenseOptionsControl.resx

```diff
@@ -1,865 +1,884 @@
+  </data>
+  <data name="_previewIntelliSenseRadioButton.Size" type="System.Drawing.Size, System.Drawing">
+    <value>143, 17</value>
+  </data>
+  <data name="_previewIntelliSenseRadioButton.TabIndex" type="System.Int32, mscorlib">
+    <value>3</value>
+  </data>
+  <data name="_previewIntelliSenseRadioButton.Text" xml:space="preserve">
+    <value>&amp;ES6 IntelliSense Preview</value>
+  </data>
+  <data name="_previewIntelliSenseRadioButton.TextAlign" type="System.Drawing.ContentAlignment, System.Drawing">
+    <value>TopLeft</value>
+  </data>
+  <data name="_previewIntelliSenseRadioButton.ToolTip" xml:space="preserve">
+    <value>Full intellisense with ES6 support* (minimum memory/CPU usage) 
+*TypeScript text editor options apply to Node.js when using ES6 preview mode.</value>
+  </data>
+  <data name="&gt;&gt;_previewIntelliSenseRadioButton.Name" xml:space="preserve">
+    <value>_previewIntelliSenseRadioButton</value>
+  </data>
+  <data name="&gt;&gt;_previewIntelliSenseRadioButton.Type" xml:space="preserve">
+    <value>System.Windows.Forms.RadioButton, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </data>
+  <data name="&gt;&gt;_previewIntelliSenseRadioButton.Parent" xml:space="preserve">
+    <value>tableLayoutPanel2</value>
+  </data>
+  <data name="&gt;&gt;_previewIntelliSenseRadioButton.ZOrder" xml:space="preserve">
+    <value>0</value>
+  </data>
+  <metadata name="intellisenseLevelGroupBox.GenerateMember" type="System.Boolean, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089">
+    <value>False</value>
+  </metadata>
+  <data name="intellisenseLevelGroupBox.AutoSize" type="System.Boolean, mscorlib">
+    <value>True</value>
+  </data>
+  <data name="tableLayoutPanel2.AutoSize" type="System.Boolean, mscorlib">
+    <value>True</value>
+  </data>
+  <data name="tableLayoutPanel2.AutoSizeMode" type="System.Windows.Forms.AutoSizeMode, System.Windows.Forms">
+    <value>GrowAndShrink</value>
+  </data>
+  <data name="tableLayoutPanel2.ColumnCount" type="System.Int32, mscorlib">
+    <value>2</value>
+  </data>
+  <data name="_analysisLogMax.Anchor" type="System.Windows.Forms.AnchorStyles, System.Windows.Forms">
+    <value>Left</value>
+  </data>
+  <data name="_analysisLogMax.Items" xml:space="preserve">
+    <value>Off</value>
+  </data>
+  <data name="_analysisLogMax.Items1" xml:space="preserve">
+    <value>100</value>
+  </data>
+  <data name="_analysisLogMax.Items2" xml:space="preserve">
+    <value>1000</value>
+  </data>
+  <data name="_analysisLogMax.Items3" xml:space="preserve">
+    <value>10000</value>
+  </data>
+  <data name="_analysisLogMax.Items4" xml:space="preserve">
+    <value>100000</value>
+  </data>
+  <data name="_analysisLogMax.Items5" xml:space="preserve">
+    <value>1000000</value>
+  </data>
+  <data name="_analysisLogMax.Items6" xml:space="preserve">
+    <value>Max</value>
+  </data>
+  <data name="
```

Comment:
How about "Learn more and send feedback" so it's more of an action?

Also now that this link is there, it's a bit difficult to group the radio buttons and link together, so adding additional margin between the radio buttons and the intellisense log entry limit would be useful.  

@billti thoughts since you commented on this earlier?